lib/capistrano/itamae/dsl.rb

```diff
@@ -52,6 +55,21 @@ def ssh_options(server)
         options << "--dry-run" if dry_run?
         options
       end
+
+      def node_options
+        return [] unless itamae_node_options.is_a? Object
+        json_file.puts(itamae_node_options.to_json)
+        json_file.close
+        ['--node-json', json_file.path]
+      end
+
+      def json_file
+        @json_file ||= if itamae_temp_node_file
+          ::Tempfile.new(['node','.json'])
```

Comment:
Please add a test case to pass this line.
Because `itamae_temp_node_file` is always false in this test suite.